### DISCUSSION TITLE
Viewing/filtering only videos in a Folder

### DISCUSSION THREAD

**Discussion by srahimeen** [photoprism/photoprism#3905] [General]
I organize my photos into folders and use the `Folders` tab a lot in PhotoPrism. However, sometimes I want to only see the videos in a specific folder.

Right now, I have to scroll through and see which thumbnails have the very light gray video/play icon on the top left in order to figure out which ones are videos.

Are there any plans to add filtering to the `Folders` view and allow for filtering for / showing just the videos in a toggle?

**Comment by graciousgrey** [comment]
A search within albums and folders is planned for the future. In the meantime you can use the search for this by searching for 
`path:your-folder-path video`

https://demo.photoprism.app/library/browse?view=cards&order=added&q=path%3AVideos%20video&public=true&quality=3

https://docs.photoprism.app/user-guide/search/filters/#filter-reference

**Comment by srahimeen** [comment]
Thanks for the info! Is there a Issue tracking this feature?

**Reply by lastzero** [reply]
As a workaround, you can sort folders (and all other types of albums) by video duration, which will then show the videos first as images have no duration:

![Screenshot 2023-12-10 at 17 13 30](https://github.com/photoprism/photoprism/assets/301686/23d019f0-d9eb-4b86-9fc6-240218c74641)

**Reply by lastzero** [reply]
The following issues seem related, although there may be more:
- https://github.com/photoprism/photoprism/issues/3767
- https://github.com/photoprism/photoprism/issues/2200
